src/new-client/src/App/TearOutSection/tearOutContext.tsx

```diff
@@ -0,0 +1,11 @@
+import { createContext } from "react"
+
+type sectionTornOut = {
+  isTornOut: boolean
+}
+
+export const sectionsTornOut: sectionTornOut = {
+  isTornOut: false,
+}
+
+export const TearOutContext = createContext(sectionsTornOut)
```

Comment:
Minor - Something more descriptive and explicit -
`createContext<TearOutContextType>(initialTearOutContext)`